Makefile

```diff
@@ -11,31 +11,36 @@ DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(she
 # but for simplicity sake we just make sure they exist in the first one, and
 # then keep using those.
 FIRST_GOPATH      ?= $(firstword $(subst :, ,$(shell go env GOPATH)))
-PROMU             ?= $(FIRST_GOPATH)/bin/promu
 GOIMPORTS         ?= $(FIRST_GOPATH)/bin/goimports
+PROMU             ?= $(FIRST_GOPATH)/bin/promu
 DEP               ?= $(FIRST_GOPATH)/bin/dep
+ERRCHECK          ?= $(FIRST_GOPATH)/bin/errcheck
 
-all: install-tools deps format build
+all: deps format build
```

Comment:
did we not want errcheck to run locally as well? This is what i expect most people to run before putting out a PR.